docs/keyconcepts.rst

```diff
@@ -0,0 +1,89 @@
+.. _keyconcepts:
+
+Key Terms used in Oppia
+========================
+
+.. admonition:: Terms
+
+   * :ref:`Topic<topic>`
+   * :ref:`Story<story>`
+   * :ref:`Skill<skill>`
+   * :ref:`Subtopic<subtopic>`
+   * :ref:`Exploration<explorations>`
+   * :ref:`Card<card>`
+   * :ref:`Content<content>`
+   * :ref:`Interaction<interaction>`
+
+.. _topic:
+
+**Topic:** A topic refers to the subject matter to be taught (e.g., Decimals, Fractions). A topic will consist of a set of stories. 
+
+.. _story:
+
+**Story:** Stories are non-contrived (realistic) scenarios that teach the topic. If the topic is Decimals, then the first story - aiming to introduce the concept - might be a scenario where a customer is in a shop and has to pay an amount in dollars and cents. A story consists of a set of :ref:`explorations <explorations>` or lessons.
+
+.. note:: 
+   Stories within a topic have to be linear, i.e., a student must complete a story before beginning the next story.
```

Comment:
This isn't true. Chapters are linear within a story, but stories can be played in any order.

Reply:
Actually, canonical stories (which is the only one we have now) are supposed to be played linearly, right?

Reply:
For reference, this is from the **Oppia Structure Project** doc where I got the information from:
 - Users will be shown a linear, ordered set of stories associated with the topic. A story must be completed before beginning the next story.
 - The story page will display the (potentially) nonlinear list of lessons that make up the narrative of the story.
Has this changed?

Reply:
Yes, I think so. I think it's reversed actually. Stories within a topic do not need to be linear. But lessons/chapters within a story are linear.

Reply:
@aks681 Canonical stories can be played/accessed in any order. We do have a suggested order (i.e. the order they are displayed on the page), but there is no obligation for learners to follow that order.